### PR TITLE
Adding documentation about our PowerShell standards, including script…

### PR DESCRIPTION
**Pull request by splatteredbits**
…s that help with applying those standards.

### CODE REVIEW

File: PowerShell/Get-LowerCaseTypeAccelerator.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<#
+#>
```
Comment (DecoyJoe) [inline PowerShell/Get-LowerCaseTypeAccelerator.ps1]: Should probably follow the standards listed in this repo itself 😆:
> Always include a synopsis, description, and at least one example in the documentation of your script.

File: PowerShell/Edit-PSFileContentTypeCase.ps1 [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    if( $typeNames.Count -eq 0 )
+    {
+        $typeNames['ordered'] = 'ordered'
+
+        [PSObject].Assembly.GetType('System.Management.Automation.TypeAccelerators')::Get.GetEnumerator() |
+            ForEach-Object { [pscustomobject]@{ Name = $_.Key ; Type = $_.Value } } |
+            Where-Object { $_.Type.Name -notlike $_.Name -or $_.Type.Namespace -ne 'System' } |
+            Where-Object { $_.Type.Name -notlike '*Attribute' } |
+            ForEach-Object { $typeNames[$_.Name] = $_.Name.ToLowerInvariant() }
```
Comment (DecoyJoe) [inline PowerShell/Edit-PSFileContentTypeCase.ps1]: Can't re-use `Get-LowerCaseTypeAccelerator.ps1` here?

File: PowerShell/Edit-PSFileContentTypeCase.ps1 [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        Write-Error -Message ('Type "{0}" not found.' -f $Name)
+        return
+    }
+
+    return $typeNames[$Name]
+}
+
+$typeNames = @{}
+
+$searchRegex = [regex]'\[([A-Za-z]+)(\[\])?\]'
```
Comment (DecoyJoe) [inline PowerShell/Edit-PSFileContentTypeCase.ps1]: Add a zero or more space patterns around the brackets? Catches code that looks like `[ string[ ] ]` ?

File: PowerShell/README.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+
+Never use `Write-Output` to return logging information. You should only ever return objects.
+
+Use `Write-Information` for progess-type logging information the user should always see.
+
+Use `Write-Verbose` for messages the user should see if they're trying to track down a problem or understand more about what your code is doing.
+
+Use `Write-Debug` for messages developer-only messages.
+
+## Script Template
```
Comment (DecoyJoe) [inline PowerShell/README.md]: This markdown isn't rendering correctly, shows up as just `## Script Template` in the rendered view.

**Comment by DecoyJoe** [comment]
@splatteredbits Bump. PR still open.

File: PowerShell/README.md (hunk 4 above)
Comment (splatteredbits) [reply PowerShell/README.md]: There was some weird character between `##` and `Script`. It's now a space.

**Pull request merged**